clmm/theory/ccl.py

```diff
@@ -67,6 +67,9 @@ def __init__(self, massdef='mean', delta_mdef=200, halo_profile_model='nfw',
                           'hernquist': {'truncated': False}}
         self.cor_factor = _patch_rho_crit_to_cd2018(ccl.physical_constants.RHO_CRITICAL)
         self.__mdelta_cor = 0.0 ## mass with corretion for input
+        self._new_version = bool(parse(ccl.__version__) >= parse('2.6'))
+        #if self._new_version:
```

Comment:
is this commented part something you forgot?

Reply:
I was going to set default alpha_ein to 0.25 (same as the NC backend), but found out it would be difficult for users to debug because the values of the relevant quantities would be different just by installing this new version. I left it there so we can uncomment it once we are ready. I can remove it if it should not be there.